Framework/Core/include/Framework/TimesliceIndex.h

```diff
@@ -30,8 +30,11 @@ struct TimesliceId {
 
 struct TimesliceSlot {
   static constexpr uint64_t INVALID = -1;
+  static constexpr uint64_t ANY = -2;
   size_t index;
   static bool isValid(TimesliceSlot const& slot);
+  bool operator==(const TimesliceSlot that) const;
```

Comment:
I think you need inline here...

Reply:
The compilation was failing because of a left semicolon in the operators definitions. AFAIK having "inline" in one place of the two is enough.